build.gradle

```diff
@@ -8,7 +8,7 @@ sourceCompatibility = 1.6
 
 group = 'com.box'
 archivesBaseName = 'box-java-sdk'
-version = '1.1.0'
+version = '1.1.1'
```

Comment:
Let's not bump the version yet, we'll do that in another commit when we release.